app/blog/_posts/2015-12-28-our-year-in-review-2015.md

```diff
@@ -0,0 +1,106 @@
+
+{% include post-image.html alt="Brasil & Iceland" src="/images/blog/year-in-review/year_start_16_9.jpg" margin="both" ratio="16_9" %}
+
+## New year, new website
+
+We’ve been running 14islands for almost four years now. The company has gone through many changes and we’ve done lots of good stuff.  Yet, we still had the same website that we built when starting the company in 2011.  It was time to do something about it, so we cleared our schedule and made it happen.
+
+{% include post-image.html alt="The new 14islands.com" src="/images/blog/year-in-review/website-1_1.png" margin="both" ratio="1_1" %}
+
+The hardest website to build is our own. It forced us to take a step back and think hard about the soul of the company. We also wanted to do a better job of showing our completed projects. After lots of discussions, post-it notes, design sessions and a trip to David's family summerhouse in [Gävle](https://www.google.se/maps/place/G%C3%A4vle/@60.6878237,17.1196136,12z/data=!3m1!4b1!4m2!3m1!1s0x4660c2ffbb3b088d:0x4034506de8c8360) - we had a new website that felt right to us.
+
+Company retreats are always a good idea to have when you have a tight team like ours. Getting close to nature fits our company spirit. And, of course, you get to drink some beers and do some nice grilling.
+
+{% include post-image.html alt="Grilled chicken" src="/images/blog/year-in-review/IMG_9047_16_9.jpg" margin="both" ratio="16_9" %}
+_Yes, that is a chicken grilling on a can of beer._
+
+
+## Awwwards in Barcelona
+
+In February, we jumped on an airplane to Barcelona to enter this years Awwwards conference. It’s an event about web design that also recognizes the most inspiring projects in the industry. It was a great experience, warmer than Stockholm, and we met tons of old friends and cool people. We wrote summaries for [day 1](http://14islands.com/blog/2015/02/27/awwwards-conference-first-day/) and [day 2](http://14islands.com/blog/2015/02/27/awwwards-conference-first-day/) at the conference.
+
+{% include post-image.html alt="Friends at Awwwards conference 2015" src="/images/blog/year-in-review/awwwards.jpg" margin="both" ratio="1_1" %}
+
+
+## Kingdom of the North
+
+One of the great things about conferences is we get time to mingle with people we’ve admired for many years. Fellow Stockholmers [North Kingdom](http://www.northkingdom.com/) were at Awwwards to accept the award “Site of the year” for the [Hobbit Chrome Experiment](http://www.northkingdom.com/cases/a-journey-through-middle-earth/).  This opportunity started the spark that lead to us helping them out on new projects. Marco collaborated with them [on a project for Disney](http://www.northkingdom.com/cases/finding-new-fantasyland/), and Hjörtur collaborated with them on [a space mission](http://www.syfy.com/theexpanse/enterthefuture/) for Syfy. Such an inspiring place, with lots of talented people.
```

Comment:
Maybe we should not link to the space missions since its IP restricted with a lame error message, how about skipping the link OR link for "Syfy" to: http://www.syfy.com/